formbricks/formbricks
# feat: Add Jost font family

## What does this PR do?

This PR adds the Jost font family to enhance the visual style of our application. The change includes updating the necessary stylesheets and components to use the new font.

Fixes #1343 

## Type of change


## How should this be tested?

- Ensure that the application displays content with the Jost font family applied.
- Check different pages/components to verify consistent font usage.
- Confirm that the font styles are responsive on various devices.

## Checklist

### Required

- [ ] First PR at Formbricks? [Please sign the CLA!](https://formbricks.com/clmyhzfrymr4ko00hycsg1tvx) Without it, we won't be able to merge it 🙏

### Appreciated
